incubator/zookeeper/values.yaml

```diff
@@ -129,6 +129,20 @@ persistence:
   accessMode: ReadWriteOnce
   size: 5Gi
 
+readinessProbe:
+  initialDelaySeconds: 20
+  periodSeconds: 30
+  timeoutSeconds: 5
+  failureThreshold: 2
+  successThreshold: 1
+
+readinessProbe:
```

Comment:
s/readinessProbe/livenessProbe on one of these

Reply:
Oh man. That's a bad one. Thanks.